boards/arm/mec2016evb_assy6797/doc/mec2016evb_assy6797.rst

```diff
@@ -0,0 +1,212 @@
+.. _mec2016evb_assy6797:
+
+MEC2616EVB_ASSY6797
+###################
+
+Overview
+********
+
+The MEC2016EVB_ASSY6797 kit is a development platform to evaluate the
+Microchip MEC1701X series microcontrollers. This board needs to be matted with
```

Comment:
I suspect this should this be ``mated`` (unless you mean is it "tangled into a thick mass" :)

Reply:
done